internal/docstore/memdocstore/mem.go

```diff
@@ -73,14 +73,17 @@ func (*URLOpener) OpenCollectionURL(ctx context.Context, u *url.URL) (*docstore.
 }
 
 // Options are optional arguments to the OpenCollection functions.
-type Options struct{}
+type Options struct {
+	// The name of the field holding the document revision.
```

Comment:
Add "Defaults to docstore.RevisionField."

Reply:
done.